content/15.development_procedure.md

```diff
@@ -66,6 +82,29 @@ The workflow for code review roughly follows this outline:
 One increasing issue with the code review process is ensuring that changes are reviewed with appropriate urgency; larger pull requests tend to languish without review, as the requirements for review necessarily add burden to the maintainers.
 "Bugfix" changes formally require only one reviewer, whereas the `yt` guidelines suggest that larger changes require review from two different team members.
 
+Recently, we have begun utilizing the ["auto-merge" feature](https://github.blog/changelog/2021-02-04-pull-request-auto-merge-is-now-generally-available/) provided by GitHub.
+This allows a maintainer to mark a pull request as "queued" to be merged once a set of requirements -- such as tests passing, approval messages, comment resolution and so forth -- are met.
+One of the most pressing bottlenecks with code review is that the time it takes the continuous integration tests to pass is much longer than the typical time span during with code review takes place.
+This results in a pull request being merged, and then whatever pull request is next to be reviewed has to wait until the tests (now updated with the newly accepted pull request) pass.
+By queuing up pull requests for merging, it allows maintainers to mark a set of pull requests as ready to be merged, and then when they meet the appropriate (automated and asynchronous) criteria, they will be merged.
+
+### Code Styling and Linting
+
+For code included in ``yt``, a set of styles are enforced.
+We use the automated code styling tool [``black``](https://black.readthedocs.io/en/stable/), which has been designed to maximize readability with as few free parameters as possible.
+(In many ways, the fact that most of the ``yt`` code developers did not utilize this style before it was enforced likely enabled its uptake, as it was seen as an choice that "made everyone compromise.")
+In addition to code formatting, other automated methods for code consistency are applied; these include ensuring that anywhere a Python "f-string" could be used it is, that all ``import`` statements are sorted (according to alphabetical order within a first/second/third-party categorization).
```

Comment:
the part on f-strings is a bit misleading. We do not use flynt anymore, but we rely on pyupgrade for string formatting among other things. Pyupgrade is however intentionally "shy" on fstring senforcement and it only uses them where it actually makes code more concise (see https://github.com/asottile/pyupgrade#f-strings)

Reply:
How's the new version?